File: 58-length-of-last-word/length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        ans = 0
        prevAns = 0
        for i in range(len(s)):
            if(s[i] == ' '):
                if(ans != 0):
                    prevAns = ans
                ans = 0
                continue
            else:
                ans += 1
        
        if(ans == 0):
            return prevAns
        else:
            return ans

File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int ans = 0;
        int prevAns = 0;

        for(char ch:s){
            if(ch == ' '){
                if(ans != 0)
                    prevAns = ans;
                ans = 0;
                continue;
            }else{
                ans++;
            }
        }

        if(ans == 0){
            return prevAns;
        }else{
            return ans;
        }
    }
};